apps/condo/domains/ticket/queries/IncidentExportTask.graphql

```diff
@@ -0,0 +1,48 @@
+query getIncidentExportTasks ($where: IncidentExportTaskWhereInput!) {
+    tasks: allIncidentExportTasks(
+        where: $where
+    ) {
+        id
+        status
+        file {
+            publicUrl
+            originalFilename
+        }
+        totalRecordsCount
+        exportedRecordsCount
+        __typename
+    }
+}
+
+mutation createIncidentExportTask ($data: IncidentExportTaskCreateInput!) {
+    task: createIncidentExportTask(
+        data: $data
+    ) {
+        id
+        status
+        file {
+            publicUrl
+            originalFilename
+        }
+        totalRecordsCount
+        exportedRecordsCount
+        __typename
+    }
+}
+
+mutation updateIncidentExportTask ($data: IncidentExportTaskUpdateInput!, $id: ID!) {
```

Comment:
why id is second argument? I thought all our API is designed to pass ID as first arg (what to change), and data is after (what exactly to change)